project/Dependencies.scala

```diff
@@ -54,13 +54,19 @@ object Dependencies {
     "com.mattbertolini" % "liquibase-slf4j" % "2.0.0"
   )
 
+  val coreDependencies = List(
+    "com.typesafe.akka" %% "akka-actor" % akkaV
+  ) ++ testDependencies
+
+  val backendDependencies = List(
+    "org.broadinstitute" %% "wdl4s" % wdl4sV
```

Comment:
Is this move right? You're saying that the engine will never read the workflow definition but the backend will?

If anything (expressions notwithstanding and I know that backend **does** need this for now) I would have thought it would be the other way round?